Source/PINDiskCache.m

```diff
@@ -16,6 +16,10 @@
 [[NSString stringWithUTF8String:__FILE__] lastPathComponent], \
 __LINE__, [error localizedDescription]); }
 
+#define PINDiskCacheException(exception) if (exception) { NSLog(@"%@ (%d) ERROR: %@", \
+[[NSString stringWithUTF8String:__FILE__] lastPathComponent], \
+__LINE__, [exception reason]); }
```

Comment:
I wonder if this should, instead of logging, `NSAssert(NO, [exception reason]);`? That way it will crash in debug but not in production (by default)? What do you think?

Reply:
Yeah I'm into that. Its a pretty serious error. I'll make the changes.